modules/directives/src/main/scala/scala/build/preprocessing/directives/UsingPackagingDirectiveHandler.scala

```diff
@@ -0,0 +1,67 @@
+package scala.build.preprocessing.directives
+
+import scala.build.EitherCps.{either, value}
+import scala.build.Logger
+import scala.build.errors.{BuildException, MalformedInputError, UnexpectedDirectiveError}
+import scala.build.options.{BuildOptions, PackageOptions, PackageType, PostBuildOptions}
+
+case object UsingPackagingDirectiveHandler extends UsingDirectiveHandler {
```

Comment:
Often the package command is used with a parameter `--option`, such like this:
```
scala-cli package --msi --output path.msi Hello.scala
```
maybe, we should also support this parameter in using directive.

Reply:
There is no mention in the documentation that scala-cli supports such a `package` directive

Reply:
More directives can be added later if needed, in subsequent PRs. And it can be mentioned in the docs when more options are supported.